signer/src/transaction_coordinator.rs

```diff
@@ -2933,4 +3028,39 @@ mod tests {
             }
         }
     }
+
+    #[tokio::test]
+    async fn test_assert_rotate_key_action_verification_window_elapsed() {
+        let context = TestContext::builder()
+            .with_in_memory_storage()
+            .with_mocked_clients()
+            .build();
+
+        let mut last_dkg: model::EncryptedDkgShares = Faker.fake();
+        last_dkg.dkg_shares_status = model::DkgSharesStatus::Unverified;
+        last_dkg.aggregate_key = public_key_from_seed(1);
+        // Set the DKG start height to be outside the verification window
+        last_dkg.started_at_bitcoin_block_height = 79u64.into(); // 11 blocks before current height of 100
```

Comment:
nit, or update comment
```suggestion
        last_dkg.started_at_bitcoin_block_height = 89u64.into(); // 11 blocks before current height of 100
```